backend/data/run.ts

```diff
@@ -27,21 +27,23 @@ db.collection('members')
       const email: string = jsonData.netid + emailDomain;
 
       const data = {
-        first_name: jsonData.firstName,
-        last_name: jsonData.lastName,
         email,
-        role: jsonData.roleId,
-        about: jsonData.about,
-        github_link: jsonData.github,
-        linkedin_link: jsonData.linkedin,
+        netid: jsonData.netid,
+        firstName: jsonData.firstName,
+        lastName: jsonData.lastName,
+        graduation: jsonData.graduation,
         major: jsonData.major,
+        doubleMajor: jsonData.doubleMajor,
         minor: jsonData.minor,
-        double_major: jsonData.doubleMajor,
+        website: jsonData.website,
+        linkedin: jsonData.linkedin,
+        github: jsonData.github,
         hometown: jsonData.hometown,
-        graduation: jsonData.graduation,
+        about: jsonData.about,
         subteam: jsonData.subteam,
-        other_subteams: jsonData.otherSubteams,
-        website: jsonData.website
+        otherSubteams: jsonData.otherSubteams,
+        role: jsonData.roleId,
+        roleDescription: jsonData.roleDescription
       };
 
       removeEmptyOrNull(data);
```

Comment:
while you are here, I will point out that `removeEmptyOrNull` only removed undefined value, but not null values

Reply:
Thank you for pointing this out! I believe this script was made by @henry-li-06 for initially adding/modifying jsons to the firebase. For this PR, I actually edited the script a little when I updated the documents in the firebase to take care of the undefined values and update fields. Should I push the updated script to git?

Reply:
Feel free to add it.